src/pages/Home/Home.page.test.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Home from './Home.page';
+
+it('renders the videos', () => {
```

Comment:
Missing the `describe` for this test case. It is a good practice to describe the set of test cases, as it is easier to understand when you have a large set of them.